packages/loopring_v3/circuit/Gadgets/AccountGadgets.h

```diff
@@ -91,27 +91,25 @@ class UpdateAccountGadget : public GadgetT {
         proofVerifierBefore(pb, TREE_DEPTH_ACCOUNTS, address,
                             leafBefore.result(), merkleRoot, proof,
                             FMT(prefix, ".pathBefore")),
-        rootCalculatorAfter(pb, TREE_DEPTH_ACCOUNTS, address,
-                            leafAfter.result(), proof,
-                            FMT(prefix, ".pathAfter")) {}
+        rootAfter(pb, TREE_DEPTH_ACCOUNTS, address, leafAfter.result(), proof,
```

Comment:
Can we also rename "address" to another name? Because address here actually means only the selection of "00 01 10 11", it's different from normal understanding.

Reply:
will rename to addressBits